addons/mail/static/src/messaging/entity/thread/thread.js

```diff
@@ -306,6 +306,31 @@ function ThreadFactory({ Entity }) {
             this.update({ areAttachmentsLoaded: true });
         }
 
+        /**
+         * Add current user to provided thread's followers.
+         */
+        async follow() {
```

Comment:
alphabetical order

Reply:
Mmh where should it go then ? It's after fetchAttachments and before isCurrentPartnerFollowing, which seems to be the right place for me, I should probably miss something

Reply:
oops, can ignore, had confusion with joinChannel in context of diff, and wrongly assumed in was non-static